bahmnicore-omod/src/main/resources/V1_100__RenameColumnPreciseInConceptNumericTable.sql

```diff
@@ -0,0 +1 @@
+alter table concept_numeric change precise allow_decimal tinyint(1);
```

Comment:
use liquibase changeset using XML instead of a query like this.

Reply:
@angshu, We inlined sql now